src/components/atoms/Modal/Modal.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, ReactNode, PropsWithChildren } from 'react'
+import * as Dialog from '@radix-ui/react-dialog'
+import classNames from 'classnames'
+import Button, { AppearanceTypes } from 'components/molecules/Button/Button'
```

Comment:
**[blocker]**

Please go over Atomic Design again, there seems to be some confusion about this in the last few PR's. 
Very briefly:

1. Atoms - basic building blocks
2. Molecules - Collections of 1 or more atoms. Usually more complex and actively used components. (For example Input component, which uses Label, input, errormessage atoms, etc)
3. Organism - Collection of 1 or more molecules and 0 or more atoms. Usually big complex components, like a Form, which essentially includes all different types of inputs + some internal logic

The following we use more freely:

4. Templates - Basically sections of pages. Contain some organisms/molecules/atoms and include the internal logic for that section, when relevant, so we can reuse them for different pages. We also use these for wrappers and layouts.
5. Pages - In a sense we have pages as well based on atomic design, but here we mostly just focus on pages of a website, so these can consist of basically anything relevant for that page.

Reply:
Totally overlooked, automatically added it to atoms..